edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Service/DependencyManager.cs

```diff
@@ -208,6 +208,7 @@ void RegisterAmqpModule(ContainerBuilder builder)
             int configUpdateFrequencySecs = this.configuration.GetValue("ConfigRefreshFrequencySecs", 3600);
             TimeSpan configUpdateFrequency = TimeSpan.FromSeconds(configUpdateFrequencySecs);
             bool checkEntireQueueOnCleanup = this.configuration.GetValue("CheckEntireQueueOnCleanup", false);
+            int messageCleanupIntervalSecs = this.configuration.GetValue("MessageCleanupIntervalSecs", -1);
```

Comment:
>-1 [](start = 103, length = 2)

what if we default the value to 1800 here?  Wouldn't that make the MessageStore logic simpler as we wouldn't need to separate hardcode another default value.

Reply:
This will change the current behavior where TTL/2 controls cleanup interval, but it doesn't matter since this env var addition will solve that control anyway